skyvern/services/pdf_import_service.py

```
@@ -133,156 +133,164 @@ def strip_prefixes(text: str) -> tuple[str, set[str]]:
+            response = self._sanitize_workflow_json(response)
+            workflow_create_request = WorkflowCreateYAMLRequest.model_validate(response)
+        except Exception as e:
+            LOG.error(
+                "Failed to validate workflow request",
+                error=str(e),
+                error_type=type(e).__name__,
+                response_sample=str(response)[:1000],
                 organization_id=organization.organization_id,
+                exc_info=True,
             )
+            raise HTTPException(status_code=422, detail=f"Failed to validate workflow structure: {str(e)}")
```

Ruff (0.14.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)